.github/workflows/rust.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       run: |
         cargo build --features storage-service
         mkdir /tmp/WORK
-        cargo run --features storage-service --bin linera -- net up --storage-config-namespace service:tcp:localhost:1235:table --policy-config devnet --path /tmp/WORK &
+        cargo run --features storage-service --bin linera -- net up --storage-config-namespace service:tcp:localhost:1235:table --policy-config devnet --path /tmp/WORK --validators 4 --shards 4 &
```

Comment:
Is `service:tcp:localhost:1235:table` supposed to be a namespace?
How can the validators share the same namespace and the same service?

Reply:
The `service:tcp:localhost:1235:table` is parsed into the following:
* `service` means that `storage-service` is being used.
* `tcp` is mandated for the protocol being used.
* `localhost` is the host being used.
* `1235` is the port being used for the `storage-service`.

Looking in an inner way, what is going on we find that `linera net up` is using the `handle_net_up_service` which itself calls the `LocalNetConfig`. Therefore, the system being used is the one  implemented for that instantiaton of the trait `LineraNetConfig`.

The implementation of the trait is in `local_net.rs` which calls the `fn initialize_storage`. The namespace being used are then specific to the validators. The key line to see is 
```
        let namespace = format!("{}_server_{}_db", self.namespace, validator);
```
In other words, the namespace being used are `table_server_X_db` with X=0, 1, 2, 3 in that case.

In summary, each validator has its namespace which are shared by all the shards of a given validator. That design is the one that we have used since about May 2023.

Reply:
I was mostly commenting on the name of the option `--storage-config-namespace`

Reply:
Ok, I can rename it as `storage`.

Reply:
--storage-config also possible